stream.go

```diff
@@ -448,8 +458,14 @@ type csAttempt struct {
 }
 
 func (cs *clientStream) commitAttemptLocked() {
-	cs.committed = true
 	cs.buffer = nil
+	if !cs.committed {
+		cs.committed = true
+		for _, rb := range cs.returnBuffers {
+			rb.Done()
+		}
+		cs.returnBuffers = nil
```

Comment:
Since we are setting this to `nil`, do we need the `if !committed` at the top?

Reply:
Correct, done.